task/jefe/src/main.rs

```diff
@@ -145,13 +152,15 @@ struct ServerImpl<'s> {
     task_states: &'s mut [TaskStatus; NUM_TASKS],
     deadline: u64,
     reset_reason: ResetReason,
+    #[cfg(feature = "dump")]
+    dump_areas: Option<u32>,
```

Comment:
If `jefe` initializes its own dump regions (per discussion [here](https://github.com/oxidecomputer/hubris/pull/1219/files#r1142706351)), could this just be a `u32`?

Reply:
Done